package.json

```diff
@@ -79,6 +78,7 @@
     "protobufjs": "^6.8.6",
     "proxyquire": "^2.0.1",
     "source-map-support": "^0.5.6",
+    "through2": "^3.0.0",
```

Comment:
There aren't that many uses in tests so we could get rid of this too, though getting it out of the production code is a win enough.

Reply:
It looks like the GAPIC generator we use under the hood is bringing it in for tests. I'm happy to go chase that down in another PR, but one step at a time :)

Just to set expectations - this module is still brought in via transitive dependencies, but again, one step at a time :)